app/src/main/java/com/CodeNaroNa/vendor/relief/FragmentsKotlin/CoronaSpreadDataFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.CodeNaroNa.vendor.relief.FragmentsKotlin
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.CodeNaroNa.vendor.relief.GlobalHelpers.Constants
+import com.CodeNaroNa.vendor.relief.databinding.FragmentCronaSpreadDataBinding
+
+
+class CoronaSpreadDataFragment : Fragment()
+{
+    private var _binding: FragmentCronaSpreadDataBinding? = null
+    private val binding get() = _binding!!
```

Comment:
This doesn't require 2 variables, merge it into one.

Reply:
Sure Ok